js/views/PageNav.js

```diff
@@ -0,0 +1,67 @@
+import electron from 'electron';
+import { View } from 'backbone';
+import loadTemplate from '../utils/loadTemplate';
+import app from '../app';
+
+const remote = electron.remote;
+
+export default class PageNav extends View {
+  constructor(options) {
+    super({
+      className: 'pageNav',
+      events: {
+        'click .js-navClose': 'navCloseClick',
+        'click .js-navMin': 'navMinClick',
+        'click .js-navMax': 'navMaxClick',
+      },
+      ...options,
+    });
+
+    this.listenTo(app.localSettings, 'change:mac_style_win_controls',
+      this.onWinControlsStyleChange);
+    this.setWinControlsStyle(app.localSettings.get('mac_style_win_controls') ? 'mac' : 'win');
+  }
+
+  setWinControlsStyle(style) {
+    if (style !== 'mac' && style !== 'win') {
+      throw new Error('Style must be \'mac\' or \'win\'.');
+    }
+
+    this.$el.removeClass('winStyleWindowControls macStyleWindowControls');
+    this.$el.addClass(style === 'mac' ? 'macStyleWindowControls' : 'winStyleWindowControls');
+  }
+
+  onWinControlsStyleChange(model, useMacStyle) {
+    this.setWinControlsStyle(useMacStyle ? 'mac' : 'win');
+  }
+
+  navCloseClick() {
+    if (remote.process.platform !== 'darwin') {
+      remote.getCurrentWindow().close();
+    } else {
+      remote.getCurrentWindow().hide();
+    }
+  }
+
+  navMinClick() {
+    remote.getCurrentWindow().minimize();
+  }
+
+  navMaxClick() {
+    if (remote.getCurrentWindow().isMaximized()) {
+      remote.getCurrentWindow().unmaximize();
+      // this.$('.js-navMax').attr('data-tooltip', window.polyglot.t('Maximize'));
+    } else {
+      remote.getCurrentWindow().maximize();
+      // this.$('.js-navMax').attr('data-tooltip', window.polyglot.t('Restore'));
+    }
+  }
+
+  render() {
+    loadTemplate('pageNav.html', (t) => {
```

Comment:
I think the chance of having templates outside of the template folder is pretty small, and that's a fine rule to have in place.